### PR TITLE
Add clothing store

### PR DESCRIPTION
**Pull request by zkdan**

### CODE REVIEW

File: scripts.js [hunk 1]
```diff
@@ -1,51 +1,51 @@
 
 // this is the store data. normally we would be grabbing data from elsewhere (an external api), but this will represent the data model! 
-  const totalInventory = [
```
Comment (zkdan) [inline scripts.js:3]: seems great

File: scripts.js [hunk 2]
```diff
@@ -1,51 +1,51 @@
...
-      stock: 4,
-    }, 
-    {
-      title: 'Don\'t Know Tee',
-      url: 'images/dontevenknow.jpg',
-      price: 22.50,
-      stock: 8,
-    }, 
-    {
-      title: 'Doughnut Jean Jacket',
```
Comment (zkdan) [inline scripts.js:17]: donut is spelled wrong!

File: scripts.js [hunk 3]
```diff
@@ -1,51 +1,51 @@
...
-      stock: 10,
-    }, 
-    {
-      title: 'HackerYou Hoodie',
-      price: 50.00,
-      stock: 4,
-    }, 
-  ]
-
-  // document ready function allows page to load before running any of our scripts
```
Comment (zkdan) [inline scripts.js:47]: we're not using jqury this time

**Review by zkdan — COMMENTED** [review]
still trying! we gotta refactor and try again.

File: scripts.js [hunk 4]
```diff
@@ -1,51 +1,51 @@
...
-    }, 
-  ]
-
-  // document ready function allows page to load before running any of our scripts 
-$(function(){
+const totalInventory = [
+  {
+    title: 'Bowie Tee',
+    url: 'images/bowie.jpg',
+    price: 19.99,
```
Comment (zkdan) [inline scripts.js:7]: this looks amazing!

**Comment by CalAkler** [comment]
This is some good code.

**Pull request merged**